src/lib/operators/aggregate_sort.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <boost/container/pmr/polymorphic_allocator.hpp>
+#include <boost/container/scoped_allocator.hpp>
+#include <boost/functional/hash.hpp>
+#include <functional>
+#include <limits>
+#include <memory>
+#include <optional>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "abstract_aggregate_operator.hpp"
+#include "abstract_read_only_operator.hpp"
+#include "expression/aggregate_expression.hpp"
+#include "resolve_type.hpp"
+#include "storage/abstract_segment_visitor.hpp"
+#include "storage/reference_segment.hpp"
+#include "storage/value_segment.hpp"
+#include "types.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+/*
+Operator to aggregate columns sort-based by certain functions, such as min, max, sum, average, and count. The output is a table
+ with reference segments. As with most operators we do not guarantee a stable operation with regards to positions -
+ i.e. your sorting order.
+
+For implementation details, please check the wiki: https://github.com/hyrise/hyrise/wiki/Aggregate-Operator
```

Comment:
The link should be https://github.com/hyrise/hyrise/wiki/Operators_Aggregate; also, the wiki page has not been updated, so this seems to be out-of-place